instrumentation/opentelemetry-instrumentation-requests/src/opentelemetry/instrumentation/requests/__init__.py

```diff
@@ -175,12 +221,16 @@ def _instrumented_requests_call(
             token = context.attach(
                 context.set_value(_SUPPRESS_HTTP_INSTRUMENTATION_KEY, True)
             )
+
+            start_time = time()
```

Comment:
Use the `default_timer` here. https://github.com/open-telemetry/opentelemetry-python-contrib/pull/1128#discussion_r898339379

Reply:
Done.